BlogTemplate/Models/ExcerptGenerator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BlogTemplate.Models
+{
+    public class ExcerptGenerator
+    {
+        private BlogDataStore _dataStore;
+
+        public ExcerptGenerator(BlogDataStore dataStore)
```

Comment:
You don't use the dataStore, so you don't need it here.  Or the rest of the constructor.